src/resources/cpp/lambdas.md

```diff
@@ -0,0 +1,365 @@
+## Abstract
```

Comment:
Every page should start with an h1, that's how pages get page titles. E.g.

![image](https://github.com/user-attachments/assets/561f4140-16cb-4261-bdf4-affddde6db42)

vs 

![image](https://github.com/user-attachments/assets/c4c4ccfa-ba31-4bf5-9df1-7a53734900f6)



I would also say let's not make an "abstract" section, this isn't a paper :)